profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -225,6 +226,20 @@ function tmgmt_poetry_form_tmgmt_job_form_alter(&$form, &$form_state, $form_id)
 }
 
 /**
+ * Custom validator to add a server validation step for dates in the past.
+ */
+function _tmgmt_poetry_validate($form, &$form_state) {
```

Comment:
Ok my idea was to have a generic name in case we add more validation afterwards, but i guess we'll just need to change the name if it happens :)